mrp_bom_product_price_margin/models/product_template.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2024 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    standard_price = fields.Float(
+        tracking=True,
```

Comment:
I think this change should go in another OCA or custom module. 
Dont think it's related to mrp. Do you ?

Reply:
My idea is : as we change a value on Product being on BoM Form view, it's great for the user to have this tracking to be sure that it "works". Other use case, you missclick on Apply price, and you want to check former price